core/src/main/java/io/seata/core/rpc/netty/RpcClientBootstrap.java

```diff
@@ -129,57 +128,29 @@ public void start() {
                     .option(EpollChannelOption.TCP_QUICKACK, true);
             }
         }
-        if (nettyClientConfig.isUseConnPool()) {
-            clientChannelPool = new AbstractChannelPoolMap<InetSocketAddress, FixedChannelPool>() {
```

Comment:
clientChannelPool is never used, except shutdown.

Reply:
Done.